### PR TITLE
Add App Store badges

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add lokalised app store badges.

## Justify why they are needed

I used the official resources from Google and Apple. Google only gives PNG and Apple offers SVG. I used the preferred color (black).

For some strange reason the English Google Play store badge has a different icons to the rest 🤷‍♂️

It would be better to load resources on demand but that's an optimization we can do later.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #409** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/409" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/409?utm_source=stack-comment).

File: apps/store/src/components/AppStoreBadge/AppStoreBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  [Locale.DaDk]: GoogleDaDk,
+  [Locale.EnDk]: GoogleEn,
+  [Locale.NbNo]: GoogleNbNo,
+  [Locale.EnNo]: GoogleEn,
+}
+
+export const AppStoreBadge = ({ type, locale }: AppStoreBadgeProps) => {
+  if (type === 'apple') {
+    const Component = APPLE_BADGE[locale]
+    return <Component />
```
Comment (simonauner) [inline apps/store/src/components/AppStoreBadge/AppStoreBadge.tsx:38]: Will there be any error if we introduce a new locale that has no badge? Will it be a runtime error or can we detect it and throw or just return null?

**Review by simonauner — APPROVED** [review]
Nice. I guess the English Google Play icon has the new Play store design, it was recently updated.

https://9to5google.com/2022/07/27/google-play-new-logo-2022/

File: apps/store/src/components/AppStoreBadge/AppStoreBadge.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/AppStoreBadge/AppStoreBadge.tsx:38]: Hmm, yeah perhaps I can add a fallback.

Usually when we add support for a new locale/country it's a big project where we go over the whole flow/code base so pretty sure we would catch this :)

Comment (simonauner) [reply apps/store/src/components/AppStoreBadge/AppStoreBadge.tsx:38]: Yeah, it's no biggie

**Pull request merged**